api/users/repositories/organizationReopository.go

```diff
@@ -0,0 +1,49 @@
+package repositories
+
+import (
+	"context"
+
+	"github.com/null-channel/eddington/api/infrastrucure"
+	models "github.com/null-channel/eddington/api/users/models"
+)
+
+type IOrgReposiotry interface {
+	Seedable
+	GetOrgByOwnerId(id string, ctx context.Context) ([]*models.Org, error)
```

Comment:
context should be first argument, In golang it's common best practice to put context as the first peram.